turborepo-tests/integration/tests/single_package/dry-run.t

```diff
@@ -19,7 +19,7 @@ Check
   Tasks to Run
   build
     Task                           = build                                                                                                                                           
-    Hash                           = d2295def33764d46                                                                                                                                
+    Hash                           = d2295def33764d46
```

Comment:
I think this is a whitespace diff? Any idea what's going on here?

Reply:
No clue, but it was causing some test failures